.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
   - 6
-script: npm run lint && npm test && npm run build
+script:
+  - npm run lint
+  - npm test
+  - npm run build
```

Comment:
I would handle it through npm scripts like
```
"pretest": "npm run lint",
"test": "..."
"posttest": "npm run build"
```

Reply:
Yea, fair!